src/Components/Web.Extensions/src/wwwroot/headManager.js

```diff
@@ -0,0 +1,110 @@
+(function () {
+    // Local helpers
+
+    const blazorIdAttributeName = '_blazor_id';
+    const headCommentRegularExpression = /\W*Head:[^{]*(.*)$/;
+    const prerenderedTags = [];
+
+    function createHeadTag({ tagName, attributes }, id) {
+        const tagElement = document.createElement(tagName);
+
+        if (id) {
```

Comment:
When is this null?

Reply:
During prerendering, ids aren't assigned to head elements because they never need to be mutated.

Reply:
I could add an inline comment to make this more clear.